# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1360457645 18000
#      Sat Feb 09 19:54:05 2013 -0500
# Node ID 85b8a1d435d65869553f435280d169c593dd21d8
# Parent  cf95f596d1fc1ca7d468c92de71b93ba2018cf47
* test/Makefile.am: Use stamp file for bc-overload-tests.

diff --git a/test/Makefile.am b/test/Makefile.am
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -66,18 +66,19 @@ test_sparse.m: build_sparse_tests.sh
 	$(srcdir)/build_sparse_tests.sh
 
 GENERATED_BC_OVERLOADS_DIRS := \
   $(shell $(srcdir)/build_bc_overload_tests.sh --list-dirs)
 
 GENERATED_BC_OVERLOADS_FILES := \
   $(shell $(srcdir)/build_bc_overload_tests.sh --list-files)
 
-$(GENERATED_BC_OVERLOADS_FILES): build_bc_overload_tests.sh bc_overloads_expected
+bc-overload-tests.stamp: build_bc_overload_tests.sh bc_overloads_expected
 	$(srcdir)/build_bc_overload_tests.sh $(srcdir)/bc_overloads_expected
+	touch $@
 
 EXTRA_DIST = \
   build_sparse_tests.sh \
   build_bc_overload_tests.sh \
   bc_overloads_expected \
   build_bc_overloads_expected.m \
   $(FCN_FILES)
 
